config/kubernetes/production/ingress.yml

```diff
@@ -21,11 +21,12 @@ metadata:
     nginx.ingress.kubernetes.io/modsecurity-transaction-id: "$request_id"
     nginx.ingress.kubernetes.io/modsecurity-snippet: |
       SecRuleEngine On
-      SecRequestBodyLimit 1048576
+      SecRequestBodyLimit 15728640
```

Comment:
Turns out I forgot to apply the fix for the file size modsec issue a while back to the production ingress file which would explain why users are reporting issues uploading some files that are deemed 'too big' 🤦‍♀️